Automattic/wp-calypso
# DIFM: Minor design updates for page picker step

#### Proposed Changes

* [Updates thumbnails for Contact, FAQ, Pricing & Team pages](https://github.com/Automattic/wp-calypso/commit/895d51eddb937ab909b46fbbd5404628fe1a981f).
* [Fixes invalid DOM nesting](https://github.com/Automattic/wp-calypso/commit/7d71a04bae383fc0fe1029128bfa1119b7225786): `div` cannot be a descendant of `p`. This fixes a console error.
* [CSS fixes for non-EN locales](https://github.com/Automattic/wp-calypso/commit/32124d6c0cac29a7dd53036fc5e94bfa2cd5cd7b)
  * Change badge width to `auto`.
  * Make the`CellLabelContainer` element a fixed width element.

#### Testing Instructions



* Go to `/start/do-it-for-me` and proceed through the steps till you reach the Page Picker step.
* Confirm that the thumbnails for the Contact, FAQ, Pricing & Team pages have been updated according to the design. (uHneAuPrNrodQAvefnzbIA-fi-3210%3A6190)
* Confirm that the Popular badge has fixed padding and auto-adjusts to the text content.
* Confirm that long page names are properly wrapped.
* Check the above on mobile & tablet screens as well.

![image](https://user-images.githubusercontent.com/5436027/196609164-50b5a4d9-05c6-49b0-93cd-a82d1961da1d.png)


#### Pre-merge Checklist






Fixes Automattic/martech#1179
Fixes Automattic/martech#1191
